redaxo/src/addons/phpmailer/CHANGELOG.md

```diff
@@ -6,8 +6,11 @@ Version 2.5.0 – XX.XX.2018
 
 ### Neu
 
+* Vendor-Update auf 6.0.6 (@skerbis)
 * E-Mail-Benachrichtigung bei Fehlern (@phoebusryan, @skerbis)
-* AutoTLS kann aktiviert/deaktiviert werden (@skerbis)
+* AutoTLS kann aktiviert/deaktiviert werden (@skerbis) 
+* UI überarbeitet und optimiert (@skerbis) 
+* Speichern und Testen (@skerbis)
```

Comment:
Changelog sollten wir rausnehmen. Das macht gregor in einem rutsch beim release

Reply:
ok. also alle Einträge für dieses Release raus?

Reply:
Wieder auf den zustand zurück wie er vor diesem PR war

Reply:
Ähm das habe ich die ganze Zeit manuell gepflegt

Reply:
habe es zurückgesetzt